<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_modificarorden</class>
 <widget class="QMainWindow" name="form_modificarorden">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>823</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Insertar Orden</string>
      </property>
      <widget class="QLineEdit" name="le_norder">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>50</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>No. de Orden</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cb_excento">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>100</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Excento de pago</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tw_platos">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>210</y>
         <width>511</width>
         <height>421</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>Plato</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Cantidad</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Precio x Unidad</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Precio Total</string>
        </property>
       </column>
      </widget>
      <widget class="QComboBox" name="cb_plato">
       <property name="geometry">
        <rect>
         <x>142</x>
         <y>160</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>160</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Plato:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>160</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Cantidad:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="sb_cantidad">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>160</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <double>1000000000.990000009536743</double>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_add">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>160</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_del">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>240</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QLabel" name="l_costo">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>660</y>
         <width>301</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Costo de la Orden</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_accep">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>700</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_cancel">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>700</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="de_fecha">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>50</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
       <property name="currentSection">
        <enum>QDateTimeEdit::DaySection</enum>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>50</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Fecha:</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>858</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
